pkg/experimental/tests/rpm/BUILD

```diff
@@ -183,7 +183,7 @@ genrule(
     for f in $(location :config_file); do
         (
             echo -n /$$(basename $$f),
```

Comment:
This might be simpler to understand as.
echo -n /$$(basename $$f),$($(execpath :md5) $$f),root,root,100644,cnm'

Or maybe not. But I think $($(execpath :md5) $$f)  might do what you want without having to fool around with the tr \\n ,

Reply:
> This might be simpler to understand as.
> echo -n /$$(basename $$f),$($(execpath :md5) $$f),root,root,100644,cnm'
> 
> Or maybe not.

This ended up working for me:

```
echo /$$(basename $$f),$$($(execpath //tests/util:md5) $$f),root,root,100644,cmn, >> $@
```

It's not bad, but it all kinda blurs together when I read it.  The multi-line one works has a similar problem, but it's easier to stick comments in there to explain empty fields or other oddities.

In an ideal world, there would be a rule that would accept `pkg_filegroup`s and emit CSV without having to write a script like this.  It's likely overkill w/o uses outside of `pkg_rpm`, though.

> But I think $($(execpath :md5) $$f) might do what you want without having to fool around with the tr \n ,

Indeed it does.  Thanks for the suggestion.

I'm going to leave this as-is for now.  I'll take a look at adjusting this in a future change.